### PR TITLE
Update the-changelog-362.md

### PR DESCRIPTION
**Pull request by anchetaWern**
Fix some unintelligibles in thechangelog episode 362

### CODE REVIEW

File: podcast/the-changelog-362.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ There's a whole domain of tools, as well as some domain areas in our industry, A
 
 **Jerod Santo:** For sure.
 
-**Amal Hussein:** And what happens with library author land is folks are really busy, they're maintainers for really large projects and they're already overburdened, and getting good documentation is a challenge that most folks have out of their projects. So kind of taking the step to democratize the power of this has kind of been left on \[unintelligible 00:27:05.11\] wider community.
+**Amal Hussein:** And what happens with library author land is folks are really busy, they're maintainers for really large projects and they're already overburdened, and getting good documentation is a challenge that most folks have out of their projects. So kind of taking the step to democratize the power of this has kind of been left on like I would say.. as a wider community.
```
Comment (pnevares) [inline podcast/the-changelog-362.md]: This sounds like _**us** as a wider community_, can you update?

File: podcast/the-changelog-362.md [hunk 2]
```diff
@@ -186,7 +186,8 @@ So you can make that decision binary, and you can enforce those things for your
 
 And lastly, what I'll say is that using ASTs is one way to really add a resilience layer to your codebase.. Because if you're fixing a bug, the first thing you should ask yourself is "Alright, I fixed this bug. Could I have avoided this with a linting rule?" If the answer is no, the next question is "Okay, could I have avoided this with a unit test?" If the answer is no, then an integration test.
 
-\[00:32:16.02\] For me, writing your own custom linting rules or custom transforms and all of these things are like a first-layer defense for a lot of things in codebases \[unintelligible 00:32:26.10\]
+\[00:32:16.02\] For me, writing your own custom linting rules or custom transforms and all of these things are like a first-layer defense for a lot of things in codebases \[
+00:32:26.10\]
```
Comment (pnevares) [inline podcast/the-changelog-362.md]: This timestamp for the resolved unintelligible can be removed as well 👍

File: podcast/the-changelog-362.md [hunk 3]
```diff
@@ -296,7 +297,7 @@ There's a culture now with some of the larger frameworks - Angular being one of
 
 So real software, software that's been out in the wild, and has had multiple developers work on it... Applications at scale have cruft. I have yet to see applications that scale that don't use multiple languages, that don't have arcane stories behind why this weird thing exists... It's like "Alright, when you open this file, you're gonna have to turn around ten times and tap your nose once." \[laughter\]
 
-**Jerod Santo:** \[unintelligible 00:59:03.19\]
+**Jerod Santo:** and every dragons...
```
Comment (pnevares) [inline podcast/the-changelog-362.md]: Sounds like "there be dragons..." to me.

**Comment by anchetaWern** [comment]
@pnevares thanks! I updated it. This is actually my first time contributing to another repo so not sure if I'm doing things right.

File: podcast/the-changelog-362.md [hunk 4]
```diff
@@ -186,7 +186,7 @@ So you can make that decision binary, and you can enforce those things for your
 
 And lastly, what I'll say is that using ASTs is one way to really add a resilience layer to your codebase.. Because if you're fixing a bug, the first thing you should ask yourself is "Alright, I fixed this bug. Could I have avoided this with a linting rule?" If the answer is no, the next question is "Okay, could I have avoided this with a unit test?" If the answer is no, then an integration test.
 
-\[00:32:16.02\] For me, writing your own custom linting rules or custom transforms and all of these things are like a first-layer defense for a lot of things in codebases \[unintelligible 00:32:26.10\]
+For me, writing your own custom linting rules or custom transforms and all of these things are like a first-layer defense for a lot of things in codebases.
```
Comment (pnevares) [inline podcast/the-changelog-362.md]:
```suggestion
\[00:32:16.02\] For me, writing your own custom linting rules or custom transforms and all of these things are like a first-layer defense for a lot of things in codebases.
```

**Comment by pnevares** [comment]
@all-contributors please add @anchetaWern for doc

**Pull request merged**